### PR TITLE
Electoral districts for upcoming "2020 Elections to the Seimas" in Lithuania

### PR DESCRIPTION
**Pull request by sguenther85**
Electoral districts for upcoming "2020 Elections to the Seimas" in Lithuania.

Here is an direct link for the elecotiral districts:
http://www.vrk.lt/documents/10180/709868/2020_rinkimu_apygardu_centrai_v1.xlsx/41202dc3-f570-4a03-aa63-67273198e273

The election is on 11th Oct. 2020 for the "Seimas" Office (Like the Parliament).
There is no senate or something similar office.

### CODE REVIEW

**Comment by sguenther85** [comment]
Hi together,

can anybody help to review it or to merge if all is fine?

Best ;)

File: identifiers/country-lt.csv [hunk 1]
```diff
@@ -0,0 +1 @@
...
ocd-division/country:lt/ed:taurages,Tauragės
ocd-division/country:lt/ed:telsiu,Telšių
ocd-division/country:lt/ed:traku-vievio,Trakų-Vievio
ocd-division/country:lt/ed:utenos,Utenos
ocd-division/country:lt/ed:verkiu,Verkių
ocd-division/country:lt/ed:vilkaviskio,Vilkaviškio
ocd-division/country:lt/ed:zemaitijos_siaurine,Žemaitijos šiaurinė
ocd-division/country:lt/ed:ziemgalos_rytine,Žiemgalos rytinė
ocd-division/country:lt/ed:ziemgalos_vakarine,Žiemgalos vakarinė
ocd-division/country:lt/ed:zirmunu,Žirmūnų
```
Comment (jdmgoogle) [inline identifiers/country-lt.csv:1]: I'm seeing this all in one line. It might be a newline termination issue between GitHub and your computer.

Comment (sguenther85) [reply identifiers/country-lt.csv]: Possible, when I wanted to edit the file here online it fit by itself. strange.
But should be look better now.

**Review by jdmgoogle — APPROVED** [review]

**Comment by sguenther85** [comment]
Anybody else time to review and/or approve? @jpmckinney or @jloutsenhizer

**Comment by jpmckinney** [comment]
Merging based on form (haven't checked content).

**Pull request merged**